app/com/gitpitch/services/ImageService.java

```diff
@@ -66,8 +67,8 @@ public String buildBackgroundOffline(String md) {
             String fragUrl = frag.substring(0, frag.indexOf("\""));
             String imageName = FilenameUtils.getName(fragUrl);
             String imageUrl = IMG_OFFLINE_DIR + imageName;
-            return buildBackground(null, imageUrl);
 
```

Comment:
The problem is, where should the background size parameter come from?  It's not available here.  You can parse it from the markdown, which is actually an easy enough thing to do with regular expressions - but far more complex than doing this simple replace.